hardhat.config.js

```diff
@@ -1,6 +1,15 @@
 require('@nomicfoundation/hardhat-toolbox');
 require('solidity-coverage');
 
+const env = process.env.ENV || 'testnet';
+const {
+  importNetworks,
+  readJSON,
+} = require('@axelar-network/axelar-contract-deployments/evm/utils');
+const chains = require(`@axelar-network/axelar-contract-deployments/info/${env}.json`);
+const keys = readJSON(`${__dirname}/info/keys.json`);
```

Comment:
```suggestion
const keys = readJSON(`${__dirname}/keys.json`);
```
no info folder in this repo